src/operator/convolution_v1-inl.h

```diff
@@ -335,10 +335,10 @@ class ConvolutionV1Op : public Operator {
                                      oshape[2] * oshape[3]);
     // param_.workspace is in elements of sizeof(DType)
     // if param_.workspace is set to zero the nstep_ equals ishape[0] (batch)
-    nstep_ = std::max<index_t>(
-        std::min(static_cast<index_t>(param_.workspace) /
-          (shape_colunit_.Size() + shape_dstunit_.Size()), ishape[0]),
-      1);
+    nstep_ = std::max(std::min(static_cast<index_t>(param_.workspace / (shape_colunit_.Size() +
```

Comment:
Just a question: why moving the index_t to inside is better?

Reply:
Because that code can't be compiled,  compiler will throw error about type mismatch. Then I rolled back before your commit, and make some change based on that. I will move index_t out in final version.